src/login/logind.c

```diff
@@ -771,7 +772,14 @@ static int manager_vt_switch(sd_event_source *src, const struct signalfd_siginfo
 
         active = m->seat0->active;
         if (!active || active->vtnr < 1) {
-                log_warning("Received VT_PROCESS signal without a registered session on that VT.");
+                int r;
+
+                log_warning("Received VT_PROCESS signal without a registered session, restoring VT.");
+
+                r = restore_terminal("/dev/tty0", true);
+                if (r < 0)
+                        log_error_errno(r, "Failed to restore VT.");
```

Comment:
as above please change this to: `log_warning_errno(r, "Failed to restore VT, ignoring: %m");